community/index.html

```diff
@@ -16,13 +16,17 @@ <h1>Community</h1>
     <h2 id="leadership">Leadership Team</h2>
     <p>The leadership team is:</p>
     <ul>
-      <li><a href="/people/AlasdairGray/">Alasdair Gray</a> (Chair), Heriot-Watt University, Edinburgh, UK</li>
-      <li><a href="/people/CaroleGoble/">Carole Goble</a>, University of Manchester, UK</li>
-      <li><a href="/people/RafaelJimenez/">Rafael C Jimenez</a>, ELIXIR-Hub, UK</li>
-      <li><a href="/people/JustinClark-Casey">Justin Clark-Casey</a> (Support tools), University of Cambridge, UK</li>
-      <li><a href="/people/LeylaGarcia">Leyla Garcia</a> (Training and Outreach), EMBL-EBI, UK</li>
-      <li><a href="/people/NickJuty">Nick Juty</a> (External research infrastructures representative), University of Manchester, UK</li>
-      <li><a href="/people/SaralaWimalaratne">Sarala Wimalaratne</a> (Engagement and Adoption), EMBL-EBI, UK</li>
+      <li><strong>Chair:</strong> <a href="/people/AlasdairGray/">Alasdair Gray</a>, Heriot-Watt University, Edinburgh, UK</li>
+      <li><strong>Support Tools:</strong> [Currently vacant]</li>
+      <li><strong>Training and Outreach:</strong> <a href="/people/LeylaGarcia">Leyla Garcia</a>, EMBL-EBI, UK</li>
+      <li><strong>Engagement and Adoption:</strong> <a href="/people/SaralaWimalaratne">Sarala Wimalaratne</a>, EMBL-EBI, UK</li>
+      <li><strong>External research infrastructures representative:</strong> <a href="/people/NickJuty">Nick Juty</a>, University of Manchester, UK</li>
+      <li><strong>Ordinary Members:</strong>
+        <ul>
+          <li><a href="/people/CaroleGoble/">Carole Goble</a>, University of Manchester, UK</li>
+          <li><a href="/people/RafaelJimenez/">Rafael C Jimenez</a>, ELIXIR-Hub, UK</li>
+        </ul>
```

Comment:
I would say Carole and Rafael are also part of the engagement and adoption, but I will leave it to them.

Reply:
I chatted with @CaroleGoble and she is happy to be without portfolio